tests/acceptance/integration/g17/Plugins/payment/Products2CheckoutCest.php

```diff
@@ -1,175 +1,175 @@
+//		$I->wantTo('Enable Plugin 2Checkout Payments in Administrator');
+//		$I->enablePlugin($this->pluginName);
+//		$I = new PluginPaymentManagerJoomla($scenario);
+//		$I->config2CheckoutPlugin($this->pluginName, $this->checkoutAccountInformation['vendorID'], $this->checkoutAccountInformation['secretWord']);
+//	}
+//
+//	/**
+//	 * @param AcceptanceTester $I
+//	 * @param $scenario
+//	 * @throws Exception
+//	 * @since    2.1.2
+//	 */
+//	public function testProductsCheckoutFrontEnd(AcceptanceTester $I, $scenario)
+//	{
+//		$I = new ConfigurationSteps($scenario);
+//		$I->cartSetting($this->cartSetting);
+//
+//		$I->wantTo('Create Category in Administrator');
+//		$I = new CategoryManagerJoomla3Steps($scenario);
+//		$I->addCategorySave($this->categoryName);
+//
+//		$I = new ProductManagerJoomla3Steps($scenario);
+//		$I->wantTo('I Want to add product inside the category');
+//		$I->createProductSaveClose($this->productName, $this->categoryName, $this->productNumber, $this->productPrice);
+//
+//		$I->wantTo('Create user for checkout');
+//		$I = new UserManagerJoomla3Steps($scenario);
+//		$I->addUser(
+//			$this->customerInformation["userName"], $this->customerInformation["password"], $this->customerInformation["email"], $this->group, $this->customerInformation["shopperGroup"],
+//			$this->customerInformation["firstName"], $this->customerInformation["lastName"], 'saveclose'
+//		);
+//
+//		$I = new CheckoutWith2Payment($scenario);
+//		$I->checkoutProductWith2Checkout($this->customerInformation["userName"], $this->customerInformation["password"], $this->checkoutAccountInformation,$this->productName, $this->categoryName, $this->customerInformation["city"]);
+//
+//		$I = new ConfigurationSteps($scenario);
+//		$I->wantTo('Check Order');
+//		$I->checkPriceTotal($this->productPrice, $this->customerInformation["firstName"], $this->customerInformation["firstName"], $this->customerInformation["lastName"], $this->productName, $this->categoryName, $this->pluginName);
+//	}
+//
+//	/**
+//	 * @param AcceptanceTester $I
+//	 * @param $scenario
+//	 * @throws Exception
+//	 * @since    2.1.2
+//	 */
+//	public function clearAllData(AcceptanceTester $I, $scenario)
+//	{
+//		$I->wantTo('Deletion of Order in Administrator');
+//		$I = new OrderManagerJoomla3Steps($scenario);
+//		$I->deleteOrder( $this->customerInformation['firstName']);
+//
+//		$I->wantTo('Delete product');
+//		$I = new ProductManagerJoomla3Steps($scenario);
+//		$I->deleteProduct($this->productName);
+//
+//		$I->wantTo('Delete Category');
+//		$I = new CategoryManagerJoomla3Steps($scenario);
+//		$I->deleteCategory($this->categoryName);
+//
+//		$I->wantToTest('Delete User');
+//		$I = new UserManagerJoomla3Steps($scenario);
+//		$I->deleteUser($this->customerInformation["firstName"]);
+//
+//		$I->wantTo('Disable Plugin');
+//		$I->disablePlugin($this->pluginName);
+//	}
+//}
```

Comment:
It is blocked by task. I will open this code when task has been fix https://redweb.atlassian.net/browse/REDSHOP-6003